java/bulk-export/bulk-export-starter/src/main/java/sleeper/bulkexport/SqsBulkExportProcessor.java

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright 2022-2024 Crown Copyright
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package sleeper.bulkexport;
+
+import com.amazonaws.services.sqs.AmazonSQS;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import sleeper.bulkexport.model.BulkExportQuery;
+import sleeper.core.properties.instance.InstanceProperties;
+import sleeper.core.properties.table.TablePropertiesProvider;
+import sleeper.core.util.ObjectFactoryException;
+
+/**
+ * Lambda to start the bulk export job.
+ */
+public class SqsBulkExportProcessor {
+    private static final Logger LOGGER = LoggerFactory.getLogger(SqsBulkExportProcessor.class);
+
+    private final InstanceProperties instanceProperties;
+    private final AmazonSQS sqsClient;
+    private final TablePropertiesProvider tablePropertiesProvider;
```

Comment:
Wiring in the SQS client here means we can't put this in the core module. It might be better to just leave this class out for now?

Reply:
Removed